pages/common/prqlc.md

```diff
@@ -0,0 +1,25 @@
+# prqlc
+
+> PRQL compiler.
+> PRQL is a modern language for transforming data - a simple, powerful, pipelined SQL replacement.
+> More information: <https://prql-lang.org>.
+
+- Run the compiler interactively:
+
+`prqlc compile`
+
+- Compile a specific `.prql` file to stdout:
+
+`prqlc compile {{path/to/file.prql}}`
+
+- Compile a `.prql` file to a `.sql` file:
+
+`prqlc compile {{path/to/source.prql}} {{path/to/target.sql}}`
+
+- Compile a query:
+
+`echo "from employees | filter has_dog | select salary" | prqlc compile`
```

Comment:
```suggestion
`echo "{{from employees | filter has_dog | select salary}}" | prqlc compile`
```
Since the queries are a placeholder here, I would suggest adding `{{` to be highlighted by our clients. I will apply it, then the PR is GTG.